modules/test/playwright/tests/configuration-admin-web/instanceSettings.spec.ts

```diff
@@ -88,8 +99,77 @@ test('LPD-35562 Enter reserved screen name', async ({page}) => {
 
 	await page.getByLabel('Last Name').fill(lastName);
 
-	await instanceSettingsPage.saveAndWaitForAlert(
-		'Error:The screen name you requested is reserved.',
-		{autoClose: false, type: 'danger'}
-	);
+	await instanceSettingsPage.saveAndWaitForAlert({
+		autoClose: false,
+		text: 'Error:The screen name you requested is reserved.',
+		type: 'danger',
+	});
+});
+
+test('LPD-38043 Assert that a site configuration overrides its instance version', async ({
```

Comment:
Assert that a configuration at the site scope can override a configuration at the instance scope.

Reply:
I added "falsy" because the original poshi test was checking for that scenario. I forgot to add that before.

> Assert that a configuration at the site scope can override a falsy configuration at the instance scope.